### ISSUE TITLE
Flying on Azuremyst Isle

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5941]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
You can fly on azuremyst isle but only near the shore and above the water right where the boat arrives and only with mounts that can be used for both ground and flying such as love rocket/invincible/headless horseman mount etc
I believe this issue has something to do with the dreanei isle being actually in outland and not in kalimdor as it is shown on the map since they all came out with tbc and you can see the loading screen of outland when teleporting with the boat from darkshore to azuremyst or when logging in on draenei isle
**Expected behaviour**: Tell us what should happen instead
obviously shouldnt be able to fly in that zone
**Steps to reproduce the problem**:

1. go to the docks where the boat should arrive in azuremyst isle
2. get on a mount that is shown as both ground and fly (love rocket for example)
3. fly around the water

**Include proofs for this behaviour**
![WoWScrnShot_122620_012107](https://user-images.githubusercontent.com/36212275/103143290-c9303180-471b-11eb-91f1-ab12b90dfeff.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rahuum** [comment]
This is undoubtedly 'wrong', but interestingly enough, this is actually blizzlike.

https://www.youtube.com/watch?v=-6DB_ElVQeE

You could fly in azuremyst isle in retail.
